packages/launcher/package.json

```diff
@@ -42,7 +42,7 @@
     "sinon-chai": "^2.8.0",
     "tslint": "5.7.0",
     "tslint-config-standard": "6.0.1",
-    "typescript": "^2.5.2",
+    "typescript": "2.5.2",
```

Comment:
fixing TypeScript, because new version that came out today (2.6.0) is breaking our TS linting